traffic_ops/app/db/migrations/202110191500000000_add_permissions.down.sql

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
I'm not sure if it's a big deal or not, but this file has a couple extra digits in the timestamp compared to prior timestamps:
```
2021100114200000_remove_user_role.down.sql
2021100114200000_remove_user_role.up.sql
202110191500000000_add_permissions.down.sql
202110191500000000_add_permissions.up.sql
```
was that intentional?

Reply:
@srijeet0406 Did you generate these with `admin` or make the filenames yourself?

Reply:
The extra digits should have made the *todb-tests* action fail (was it even triggered on this PR?):
https://github.com/apache/trafficcontrol/blob/631d8793eb7e83f9db2624f1cc0723f9f2d4ff28/.github/actions/todb-tests/entrypoint.sh#L86-L96

Reply:
I named them myself, I wasn't even aware that I could generate them using `admin`. I'll create a follow up PR.

Reply:
> was it even triggered on this PR?

Yeah, this is the PR where we had to stop checking for being able to run `create_tables` twice, remember? The problem is that `VIOLATING_FILES` isn't being set properly, the set is empty after that `sed` call:

```console
$ ls | sort | cut -d _ -f 1 | sed -n -e '/2020061622101648/,$p' 
$
```

Reply:
Weird that it doesn't show up in the list of actions run above. It shows up when [searching runs by branch `CDN-13621`](https://github.com/apache/trafficcontrol/actions/workflows/traffic.ops.database.yml?query=branch:CDN-13621), though.